### PR TITLE
[table service] cleanup : cleanup protobuf definitions 

### PR DESCRIPTION
**Pull request by sijie**
Descriptions of the changes in this PR:

*Motivation*

There are some unused fields and some fields that were not renamed correctly from col to namespace.

*Solution*

This is a cleanup PR to remove unused fields and rename "col" to "ns" or "namespace".

### CODE REVIEW

File: stream/proto/src/main/proto/stream.proto [hunk 1]
```diff
@@ -85,6 +85,15 @@ message ParentRangesList {
 // Stream
 //
 
+// since stream and table are similar and exchangable,
+// from the beginning, we shared the metadata management
+// between streams and tables and distinguish them using
+// a flag that recorded in metadata.
+enum StreamType {
```
Comment (eolivelli) [inline stream/proto/src/main/proto/stream.proto]: Streamtype = table seems a bit weird.

What about something like StorageMode, DataModel....

Comment (sijie) [reply stream/proto/src/main/proto/stream.proto]: How about StorageType?

Comment (eolivelli) [reply stream/proto/src/main/proto/stream.proto]: Okay. Sounds good.

Comment (sijie) [reply stream/proto/src/main/proto/stream.proto]: done

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
retest this please

**Pull request closed without merging**